src/org/ods/component/Context.groovy

```diff
@@ -577,7 +577,7 @@ class Context implements IContext {
 
     private String retrieveGitCommitMessage() {
         script.sh(
-            returnStdout: true, script: 'git log -1 --pretty=%B HEAD',
+            returnStdout: true, script: "git log -1 --format='%f' HEAD",
```

Comment:
Sorry to ask, but would you mind taking this out? It's very hard to find this change in the PR here, as it is completely unrelated. I'd rather have it separately so that it is easy to link to from the changelog, and easy to revert if need be.

Reply:
@clemensutschig I can also do that if you want me to do it ...

Reply:
downvote - it's linked .. and I'll backport in one shot ... ok :)?